### DISCUSSION TITLE
Disculpen Hoy sábado 8 de octubre hay clase

### DISCUSSION THREAD

**Discussion by geraodst117** [Launch-X-Latam/MisionFrontEnd#176] [Q&A]
Cómo el título lo indica y disculpen es que no puedo acceder al calendario

**Comment by Amhedriel** [comment]
Buenas

Te recomiendo que veas los videos de las anteriores clases primero, porque dijeron un par de veces que son 2 clases a la semana, Miércoles y Viernes desde las 21:00 hora Argentina, saludos.

**Reply by geraodst117** [reply]
Muy amable y disculpen

**Reply by xXleonelXxD** [reply]
Disculpa como puedo acceder a los vídeos?

**Comment by samujm** [comment]
![MicrosoftTeams-image](https://user-images.githubusercontent.com/69430694/194779652-d3fb2c42-307e-415a-8679-b6e14ac8a01d.png)
